iwf-sdk.yaml

```diff
@@ -546,7 +546,7 @@ components:
     IDReusePolicy:
       type: string
       enum:
-        - ALLOW_IF_PREVIOUS_EXISTS_ABNORMALLY
+        - ALLOW_IF_PREVIOUS_EXITS_ABNORMALLY
```

Comment:
This would become a breaking change . I think we need to make it backward compatible